src/test/java/co/rsk/federate/signing/hsm/advanceBlockchain/HsmBookkeepingClientImplTest.java

```diff
@@ -0,0 +1,458 @@
+package co.rsk.federate.signing.hsm.advanceblockchain;
```

Comment:
Update this, it does not correspond to the path because you used a camel case for the package name. I think you should use lowercase only for the package name

Reply:
It has been updated but SonarCloud keeps complaining. Might be a bug from sonarcloud